polkadot/xcm/pallet-xcm-benchmarks/src/generic/benchmarking.rs

```diff
@@ -154,119 +187,145 @@ benchmarks! {
 			call: double_encoded_noop_call,
 		};
 		let xcm = Xcm(vec![instruction]);
-	}: {
-		executor.bench_process(xcm)?;
-	} verify {
+		#[block]
+		{
+			executor.bench_process(xcm)?;
+		}
 		// TODO Make the assertion configurable?
+
+		Ok(())
 	}
 
-	refund_surplus {
+	#[benchmark]
+	fn refund_surplus() -> Result<(), BenchmarkError> {
 		let mut executor = new_executor::<T>(Default::default());
 		let holding_assets = T::worst_case_holding(1);
 		// We can already buy execution since we'll load the holding register manually
 		let asset_for_fees = T::fee_asset().unwrap();
-		let previous_xcm = Xcm(vec![BuyExecution { fees: asset_for_fees, weight_limit: Limited(Weight::from_parts(1337, 1337)) }]);
+		let previous_xcm = Xcm(vec![BuyExecution {
+			fees: asset_for_fees,
+			weight_limit: Limited(Weight::from_parts(1337, 1337)),
+		}]);
 		executor.set_holding(holding_assets.into());
 		executor.set_total_surplus(Weight::from_parts(1337, 1337));
 		executor.set_total_refunded(Weight::zero());
-		executor.bench_process(previous_xcm).expect("Holding has been loaded, so we can buy execution here");
+		executor
+			.bench_process(previous_xcm)
+			.expect("Holding has been loaded, so we can buy execution here");
 
 		let instruction = Instruction::<XcmCallOf<T>>::RefundSurplus;
 		let xcm = Xcm(vec![instruction]);
-	} : {
-		let result = executor.bench_process(xcm)?;
-	} verify {
+		#[block]
+		{
+			let result = executor.bench_process(xcm)?;
```

Comment:
```suggestion
			let _result = executor.bench_process(xcm)?;
```

Unused variable should be prefixed with underscore